src/System.Text.Json/tests/JsonArrayTests.cs

```diff
@@ -430,20 +430,22 @@ public static void TestInsert()
         [Fact]
         public static void TestHeterogeneousArray()
         {
-            var mixedTypesArray = new JsonArray { 1, "value", true };
+            var mixedTypesArray = new JsonArray { 1, "value", true, null, 2.3, new JsonObject() };
 
             Assert.Equal(1, mixedTypesArray[0]);
             Assert.Equal("value", mixedTypesArray[1]);
             Assert.Equal(true, mixedTypesArray[2]);
+            Assert.IsType<JsonNull>(mixedTypesArray[3]);
+            Assert.Equal(2.3, mixedTypesArray[4]);
+            Assert.IsType<JsonObject>(mixedTypesArray[5]);
 
-            mixedTypesArray.Add(17);
+            mixedTypesArray.Add(false);
```

Comment:
According to https://github.com/dotnet/corefx/pull/40673#discussion_r321950019, I wanted to test more cases, so I added more types of elements to line 433 and here changed 17 to `false`, because integer was already above.